ml/base/config.py

```diff
@@ -38,6 +39,27 @@ def from_dict(cls, dic):
     def from_internal_file(cls, config_name):
         pass
 
+    @classmethod
+    def override_with_flag(cls, args_string):
+	args_list=args_string.split('--')
+	
+	#Go through all the flags.
+	for attr_arg in args_list:
+		
+		single_arg=arg.split(' ')
+		attr_name=single_arg[0].split('__')
+		attr_val=single_arg[1]
+		#For each flag, call the recursive function to populate values to the subfield.
+		populate_attr(cls,0,attr_name,attr_val)
+
+	def populate_attr(cls,i,attr_name,attr_val):
+		if i==len(attr_name)-1:
+			__getattr__(cls,attr_name[i])
```

Comment:
use `getattr`, not `__getattr__`, the underscored one is a method bound to object class.